.travis.yml

```diff
@@ -274,6 +274,12 @@ matrix:
         - ! git grep -q --ignore-case "gnu general public";
         - ! git grep -q --ignore-case "gnu library general public";
 
+    - env:
+        - NAME=include_check
+      script:
+        - echo 'Checking that there are no '#include "mbed.h"' in code where it should not be'
+        - ! git grep '^#include "mbed.h"$' -- '*.c' '*.h' '*.cpp' '*.hpp' ':!*platform_mbed.h' ':!*TESTS/*' ':!TEST_APPS/' ':!UNITTESTS/' ':!*tests/*' ':!*targets/*' ':!*TARGET_*' ':!*unsupported/*'
```

Comment:
why is `targets` folder also excluded?

Reply:
According to @AnttiKauppila: targets folder is excluded for now because it has not been fixed as of yet.

Reply:
I checked and there are 5 #mbed.h" includes under targets/. Maybe we should raise minor defects out of those. I want that corresponding teams fixes those because testing changes can be difficult.

Reply:
That explains it, lets create issues for these and will be fixed